src/Configuration/Config.php

```diff
@@ -17,4 +17,9 @@ public function getDatabases(): array
     {
         return $this->getArrayValue(['parameters', 'migrateDatabases']);
     }
+
+    public function getUsers(): array
+    {
+        return $this->getArrayValue(['parameters', 'users']);
+    }
```

Comment:
Useři nejsou v konfigu - měly by to IMHO být useři (kromě ACCOUNTADMINA) kteří dostávají nějaké z resourců, které migruješ. Ty nevíš jaké uživatele workspaců budeš migrovat... např.

Reply:
tohle jsem spíš myslel jako hesla uživatelů pro workspaces... když heslo zadáš tak se vytvoří s daným heslem a když ho nezadáš tak si ho budeš muset zadat při prvním lognutí

Reply:
Aha, jo to by asi šlo.... :) Ty hesla se budou muset celkově ještě dořešit. My je totiž v connection nijak neukládáme. Prostě je ukážeme v první response a nikde si je neuložíme. Hesla pro uživatelské workspacy si drží sandboxes api... ale to zase nemá žádné API na to aby je setlo těm userům ve SNFLK.... Možná bude potřeba aby se ten migrační skript poptal sandboxes API... Případně mu je budeme tady muset připravit nějakým skriptem.